src/index.js

```diff
@@ -8,6 +8,10 @@ window.jQuery = $; window.$ = $;
 const breakpointDesktop = 992;
 
 $(document).ready(() => {
+  $('.menu-toggle').click(() => {
+    $('.nav-mobile').toggleClass('show');
+  });
+
   if (window.innerWidth >= breakpointDesktop) {
     // for development purposes only:
     setTimeout(() => setHeights(), 500);
```

Comment:
Refs:
https://www.smashingmagazine.com/2016/06/battling-bem-extended-edition-common-problems-and-how-to-avoid-them/
http://getbem.com/naming/